webapp/sources/ldap-inventory/inventory-repository/src/main/resources/ldap/inventory.schema

```diff
@@ -66,6 +66,13 @@ attributetype ( InventoryAttributes:1.5
   DESC 'Unique identifier for the policy server of a node'
   SUP uuid )
 
+# Revision identifier is not used in these objects, but they will be at
```

Comment:
you are not using it in the inventory, did you put it there in case nodes are versionned also in the future?

Reply:
they will, it's the only way to have versioned node properties. Same for groups. It leads to a lot of complicated question regarding external providers (for ex, what does it means for a datasource property to be versioned?)